plugins/woocommerce/includes/tracks/events/class-wc-products-tracking.php

```diff
@@ -138,15 +138,31 @@ public function track_product_updated_client_side( $post ) {
 						return;
 					}
 
+					var tagsText = $( '[name=\"tax_input[product_tag]\"]' ).val();
 					var currentStockValue = $( '#_stock' ).val();
 					var properties = {
+						attributes:				$( '.woocommerce_attribute' ).length,
+						categories:				$( '[name=\"tax_input[product_cat][]\"]:checked' ).length,
+						'cross-sells':			$( '#crosssell_ids option' ).length ? 'Yes' : 'No',
+						description:			$( '#content' ).val() ? 'Yes' : 'No',
+						enable_reviews:			$( '#comment_status' ).is( ':checked' ) ? 'Yes' : 'No',
+						is_virtual:				$( '#_virtual' ).is( ':checked' ) ? 'Yes' : 'No',
+						is_downloadable:		$( '#_downloadable' ).is( ':checked' ) ? 'Yes' : 'No',
+						manage_stock:			$( '#_manage_stock' ).is( ':checked' ) ? 'Yes' : 'No',
+						menu_order:				$( '#menu_order' ).val() ? 'Yes' : 'No',
+						product_gallery:		$( '#product_images_container .product_images > li' ).length,
+						product_image:			$( '#_thumbnail_id' ).val() > 0 ? 'Yes' : 'No',
 						product_type:			$( '#product-type' ).val(),
-						is_virtual:				$( '#_virtual' ).is( ':checked' ) ? 'Y' : 'N',
-						is_downloadable:		$( '#_downloadable' ).is( ':checked' ) ? 'Y' : 'N',
-						manage_stock:			$( '#_manage_stock' ).is( ':checked' ) ? 'Y' : 'N',
-						stock_quantity_update:	( initialStockValue != currentStockValue ) ? 'Y' : 'N',
+						purchase_note:			$( '#_purchase_note' ).val() ? 'yes' : 'no',
```

Comment:
This one appears to be working correctly for me, but probably also worth checking `length` here.